test/tpc_benchmark_tests.cpp

```diff
@@ -0,0 +1,239 @@
+#include "thirdparty/microtest/microtest.h"
+
+#include "SQLParser.h"
+#include "util/sqlhelper.h"
+
+#include "sql_asserts.h"
+
+#include <string>
+#include <fstream>
+#include <streambuf>
+#include <iostream>
+
+using namespace hsql;
+
+std::string readFileContents(std::string file_path) {
+  std::ifstream t(file_path.c_str());
+  std::string text((std::istreambuf_iterator<char>(t)),
+                   std::istreambuf_iterator<char>());
+  return text;
+}
+
+int testSQLFiles(std::string sqlDatasetName, std::vector <std::string> files) {
+
+  int testsFailed = 0;
+  std::string concatenated = "";
+  for (const std::string &file_path : files) {
+    std::string query = readFileContents(file_path);
+
+    concatenated += query;
+    if (concatenated.back() != ';') concatenated += ';';
```

Comment:
Doc what you are doing here?

Reply:
No clue! But as far as I can tell, it works! 😃 
I just wrapped the old code within `testSQLFiles()`. So I'm not responsible for any of the magic here..

Reply:
As far as I understand it, it also takes all sql files, builds a single string from it, and tries to parse that. Verifying that multiple statements in a single string work should be done in a separate test. I'd remove it.

Reply:
Ok. I'll look into that next week.